### PR TITLE
OSDOCS-3079: Add the Spot instances usage to the Managing worker nodes section

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-3079

### CODE REVIEW

**Comment by bmcelvee** [comment]
@bysnupy would you mind reviewing this PR, please? Thanks!

**Comment by bmcelvee** [comment]
@xueli181114 would you mind giving this PR a QE review, please? Thank you!

File: modules/rosa-using-spot-instance.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-using-spot-instance_{context}"]
+= Adding a machine pool with spot instances to a cluster
+
+After a machine pool is created, you can configure additional machine pools with spot instances.
```
Comment (0kashi) [inline modules/rosa-using-spot-instance.adoc:9]: Creating a machine pool first is not necessary.  Defining the use of spot instances is a part of the machine pool creation process.  You might be meaning to say "Additional machine pools can be created using spot instances".

File: modules/rosa-using-spot-instance.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+----
+$ rosa create machinepool --help | grep spot
+  # Add a machine pool with spot instances to a cluster
+  rosa create machinepool -c mycluster --name=mp-1 --replicas=2 --instance-type=r5.2xlarge --use-spot-instances \
+    --spot-max-price=0.5
+      :
+      --spot-max-price string   Max price for spot instance. If empty use the on-demand price. (default "on-demand")
+      --use-spot-instances      Use spot instances for the machine pool.
+----
+endif::[]
```
Comment (0kashi) [inline modules/rosa-using-spot-instance.adoc:26]: The command is not this whole thing.  This is just showing that the spot flags exist in the `create machine pool` command.  

We should be saying that you add the additional flags to use spot instance.  We can say something like:
In order to use spot instances with your machinepool you must pass in the `--use-spot-instances` flag and if you want to define a max price you can add the `--spot-max-price` flag.  An example is:

rosa create machinepool --cluster=<cluster-name> \
--name=<machinepool-name> \
--replicas=<number-nodes> \
**--use-spot-instances \
--spot-max-price=0.5**

File: modules/rosa-using-spot-instance.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+////
+ifdef::openshift-dedicated[]
```
Comment (bysnupy) [inline modules/rosa-using-spot-instance.adoc:29]: The OCM operation to configure the spot instance is  the same between OSD and ROSA, but it looks like the OCM procedure is shown only for OSD by this condition. Could you please display this OCM procedure to both products ?

File: modules/rosa-using-spot-instance.adoc (hunk 1 above)
Comment (xueli181114) [reply modules/rosa-using-spot-instance.adoc:9]: I guess `After a machine pool is created` should be `After a cluster is created`

File: modules/rosa-using-spot-instance.adoc [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+To configure a machine pool with a spot instance:
+
+. Select the *Machine pools* tab.
+. Click the *Add machine pool* button on your OSD cluster page in OCM.
+. Add the machine pool with the spot instance by checking the *Use Amazon EC2 Spot Instances* check box.
```
Comment (xueli181114) [inline modules/rosa-using-spot-instance.adoc:34]: Lack of the steps for price settings and the last step click `Add machine pool` button.

**Comment by bmcelvee** [comment]
Once https://github.com/openshift/openshift-docs/pull/40901 is merged, I will update the OSD-specific assembly with the UI procedure.

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.11` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.10. And any PR going into `main` must also target the latest version branch (enterprise-4.11).

If the update in your PR does NOT apply to version 4.11 onward, please retarget this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by bmcelvee** [comment]
Closing this PR in favor of https://github.com/openshift/openshift-docs/pull/43045.

**Pull request closed without merging**